### DISCUSSION TITLE
spring-session-hazelcast serialization by custom serializer

### DISCUSSION THREAD

**Discussion by vpsofts** [community/community#49490] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi, please can somebody explain me idea of [spring-session-hazelcast](https://github.com/spring-projects/spring-session/tree/2.7.x/spring-session-hazelcast), directly principe of Hazelcast4SessionUpdateEntryProcessor? I have my application, i am using hazelcast for distribute computing and cache, and now i use for server session replication. For all my model object which are in session have a custom data serializers for best replication performance. But spring-session-hazelcast uses Hazelcast4SessionUpdateEntryProcessor and for serialization him use simple java serialization. And add custom serilization for this class is not possible because she has not getter for props :( I need best performance choise for data replication, but i dont want fork or custom implementation. Why Hazelcast4SessionUpdateEntryProcessor, which is holder of ALL session and hazeclast backups him / sends to another node cant serialize by custom serilizer?

Thank you very much, and sorry for my english

Petr

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
